### ISSUE TITLE
4.8.1 Segmentation fault

### ISSUE THREAD

**Issue by tomsommer** [lavv17/lftp#393]
```
umask(022)                              = 022
umask(022)                              = 022
ioctl(0, TIOCGPGRP, [20908])            = 0
getpgrp()                               = 20908
ioctl(1, TIOCGWINSZ, {ws_row=86, ws_col=317, ws_xpixel=0, ws_ypixel=0}) = 0
write(1, "                                "..., 128                                                                                                                                ) = 128
)                       = 1
write(4, "QUIT\r\n", 6)                 = 6
select(5, [4], [], NULL, {0, 63020})    = 1 (in [4], left {0, 60243})
read(4, "221  \r\n", 65536)             = 7
close(4)                                = 0
rt_sigprocmask(SIG_BLOCK, [CHLD], NULL, 8) = 0
brk(0xffd000)                           = 0xffd000
--- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=0x145a8d0} ---
+++ killed by SIGSEGV +++
Segmentation fault

```

**Comment by tomsommer** [comment]
```
(gdb) bt full
#0  0x00000000004a8d65 in Timer::~Timer() ()
No symbol table info available.
#1  0x00000000004efd1a in xmap_p<NetAccess::SiteData>::~xmap_p() ()
No symbol table info available.
#2  0x00007ffff5f4a992 in exit () from /lib64/libc.so.6
No symbol table info available.
#3  0x00007ffff5f33d24 in __libc_start_main () from /lib64/libc.so.6
No symbol table info available.
#4  0x000000000043e1b9 in _start ()
No symbol table info available.

```

**Comment by lavv17** [comment]
should be finally fixed by 6aa09a58f2cb25995f80469001bbcc477b700862
